lircd.te

```diff
@@ -28,6 +28,8 @@ allow lircd_t self:process signal;
 allow lircd_t self:fifo_file rw_fifo_file_perms;
 allow lircd_t self:tcp_socket { accept listen };
 allow lircd_t self:netlink_kobject_uevent_socket create_socket_perms;
+allow lircd_t passwd_file_t:file { read getattr open };
```

Comment:
There should be 

auth_read_passwd(lircd_t) 

rule instead of this one. Basically we try to find proper interfaces if possible, especially if we try to access private types from another modules.